src/test/TEST_ij/solvers.saved.arcticus

```diff
@@ -134,8 +134,8 @@ Iterations = 19
 Final Relative Residual Norm = 3.502389e-09
 
 # Output file: solvers.out.107
-Iterations = 34
-Final Relative Residual Norm = 8.725157e-09
+Iterations = 29
```

Comment:
Convergence improved a lot here. Was there a bug before?

Reply:
Yes, I think there was a previous bug, though I don't think it was fixed in this PR. When I reran master on this test I got the same (29 iteration) result shown here. So I guess I just hadn't updated the .saved file previously.